packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1469,9 +1469,7 @@ module.exports = (
             // backend. Discussion in:
             //
             // https://github.com/mozilla/fxa/issues/1808
-            subscriptions: isA
-              .array()
-              .items(validators.subscriptionsSubscriptionValidator),
+            subscriptions: validators.subscriptionsSubscriptionListValidator.optional(),
```

Comment:
It would make for a more uniform API if we returned the empty array instead of `undefined` in the fallback case, I think. Then consumers can always operate on an array, without having to check existence.

I thought that was what I meant to do originally as it happens, although I can't see the code now so I guess I forgot about the actual doing part.

Reply:
I had done that on my first round through this fix, but then figured we might not want subscriptions to show up in prod yet if they're not enabled server-side

Reply:
But then again, we'll want subscriptions enabled eventually, so maybe an empty list is just better going forward

Reply:
Ah yeah, I dunno, whichever you think makes more sense.